db-service/lib/cqn2sql.js

```diff
@@ -311,8 +311,8 @@ class CQN2SQLRenderer {
    */
   column_expr(x, q) {
     if (x === '*') return '*'
-    
-    let sql = this.expr({ param: false, __proto__: x })
+  
+    let sql = x.param !== true && (typeof x.val === 'number' || x.val === null) ? this.expr({ param: false, __proto__: x }): this.expr(x)
```

Comment:
Why the `null` it doesn't have a type. So whatever type the `?` is assigned by the database `null` is accepted and will be returned correctly as a `null`.

Reply:
In the case  `{ val: null, as: 'Null' },` it doesn't have a type. See [select with column aliases](https://github.com/cap-js/cds-dbs/blob/13ece9e29ea17c3ac52f36afd4e4932986bbb27d/db-service/test/cqn2sql/select.test.js#L463).

Reply:
@BobdenOs  Do you think the generated snapshot is wrong and in this case there should be a placeholder for `null`

Reply:
When changing behavior all snapshots are wrong by default. It is required to check the difference between the snapshots when changing the behavior. I would rather look at the integration tests whether the behavior is correct. If Postgres and HANA don't have problem with handling placeholders for `null` values in the select columns. I would interpretate `null` as placeholders.

Reply:
> It is required to check the difference between the snapshots when changing the behavior.
I checked this , the question is if the old result of the test is correct, with no placeholder for `null `value.
If you say it is not the problem for db to have a `null` placeholder, the test before was wrong!